tasks/customaction.py

```diff
@@ -52,17 +56,18 @@ def build(ctx, vstudio_root=None, arch="x64"):
         if arch == "x86":
             batchfile = "vcvars32.bat"
         vs_env_bat = '{}\\VC\\Auxiliary\\Build\\{}'.format(vsroot, batchfile)
-        cmd = 'call \"{}\" && msbuild omnibus\\resources\\agent\\msi\\cal\\customaction.vcxproj /p:Configuration=Release /p:Platform={}'.format(vs_env_bat, arch)
+        cmd = 'call \"{}\" && msbuild omnibus\\resources\\agent\\msi\\cal\\customaction.vcxproj /p:Configuration={} /p:Platform={}'.format(vs_env_bat, configuration, arch)
     else:
-        cmd = 'msbuild omnibus\\resources\\agent\\msi\\cal\\customaction.vcxproj /p:Configuration=Release /p:Platform={}'.format(arch)
+        cmd = 'msbuild omnibus\\resources\\agent\\msi\\cal\\customaction.vcxproj /p:Configuration={} /p:Platform={}'.format(configuration, arch)
 
     cmd += verprops
     print("Build Command: %s" % cmd)
 
     ctx.run(cmd)
-
+    srcdll = None
     if arch is not None and arch == "x86":
-        shutil.copy2("omnibus/resources/agent/msi/cal/release/customaction.dll", BIN_PATH)
+        srcdll = "omnibus/resources/agent/msi/cal/{}/customaction.dll".format(configuration)
     else:
-        shutil.copy2("omnibus/resources/agent/msi/cal/x64/release/customaction.dll", BIN_PATH)
+        srcdll = "omnibus/resources/agent/msi/cal/x64/{}/customaction.dll".format(configuration)
```

Comment:
the resulting path will have a different case than previously (`release` vs `Release`), can we always assume we're on a case-insensitive filesystem?

Reply:
yes, we can.  And, it will match the profile name, so this is more correct.